### ISSUE TITLE
Mattermost 4.0.x crashes with "panic: assignment to entry in nil map"

### ISSUE THREAD

**Issue by j-p-k** [mattermost/platform#7178]
#### Summary

Mattermost application crash

```
11:29:00
panic: assignment to entry in nil map
11:29:00
goroutine 6484 [running]:
11:29:00
github.com/mattermost/platform/vendor/github.com/nicksnyder/go-i18n/i18n/bundle.(*Bundle).translate(0xc420015a10, 0xc4221384e0, 0x122e8b8, 0x1e, 0xc421de3e40, 0x2, 0x2, 0xc420027400, 0xc4220f5e70)
11:29:00
/var/lib/jenkins/jobs/msr/jobs/d/jobs/enterprise-release/workspace/src/github.com/mattermost/platform/vendor/github.com/nicksnyder/go-i18n/i18n/bundle/bundle.go:371 +0x124
11:29:00
github.com/mattermost/platform/vendor/github.com/nicksnyder/go-i18n/i18n/bundle.(*Bundle).TfuncAndLanguage.func1(0x122e8b8, 0x1e, 0xc421de3e40, 0x2, 0x2, 0x40f0af, 0x107e600)
11:29:00
/var/lib/jenkins/jobs/msr/jobs/d/jobs/enterprise-release/workspace/src/github.com/mattermost/platform/vendor/github.com/nicksnyder/go-i18n/i18n/bundle/bundle.go:311 +0x65
11:29:00
github.com/mattermost/platform/utils.TfuncWithFallback.func1(0x122e8b8, 0x1e, 0xc421de3e40, 0x2, 0x2, 0x1a, 0xc421423e60)
11:29:00
/var/lib/jenkins/jobs/msr/jobs/d/jobs/enterprise-release/workspace/src/github.com/mattermost/platform/utils/i18n.go:124 +0x5f
11:29:00
github.com/mattermost/platform/app.UpdateLastActivityAtIfNeeded(0xc4222c29a0, 0x1a, 0xc4222c29c0, 0x1a, 0x15dd0e0b247, 0x15e6b5f7a47, 0x15dd10623b5, 0xc4222c29e0, 0x1a, 0x0, ...)
11:29:00
/var/lib/jenkins/jobs/msr/jobs/d/jobs/enterprise-release/workspace/src/github.com/mattermost/platform/app/session.go:197 +0x226
11:29:00
github.com/mattermost/platform/app.NewWebConn.func1(0xc4226ea480)
11:29:00
/var/lib/jenkins/jobs/msr/jobs/d/jobs/enterprise-release/workspace/src/github.com/mattermost/platform/app/web_conn.go:48 +0x82
11:29:00
created by github.com/mattermost/platform/app.NewWebConn
11:29:00
/var/lib/jenkins/jobs/msr/jobs/d/jobs/enterprise-release/workspace/src/github.com/mattermost/platform/app/web_conn.go:49 +0x251
```

This was first observed with Mattermost 4.0.2 Enterprise Edition. Upgrade to 4.0.3 and downgrade to 4.0.0 did not solve the issue.

We had previously run Mattermost 4.0.0 without issues. This behaviour was first observed after upgrade to 4.0.2

#### Steps to reproduce

On our system the crash occurs 20 minutes after starting the Mattermost application. We use a MySQL database with 50 users. Mattermost was upgraded multiple times in the past. 

The language used by the system and all users is English.

**Comment by enahum** [comment]
Do you have an existant i18n folder in your Mattermost installation? Looks like the folder might be missing?

**Comment by j-p-k** [comment]
@enahum Yes, the folder exists.

We observed the reported behaviour on a machine that was overloaded with webhook requests at the time. We had requests and database queries timing out.

The behaviour could not be reproduced after the load went back to normal.

I'm leaving this issue open in case someone wants to investigate.

**Comment by enahum** [comment]
@j-p-k  #7282 will fix this and It should be available in 4.2
